pkgs/top-level/all-packages.nix

```diff
@@ -20447,7 +20447,9 @@ in
 
   wavebox = callPackage ../applications/networking/instant-messengers/wavebox { };
 
-  sonic-pi = libsForQt5.callPackage ../applications/audio/sonic-pi { };
+  sonic-pi = libsForQt5.callPackage ../applications/audio/sonic-pi {
+    ruby = ruby_2_4;
```

Comment:
I think this pinning deserves a comment for context for the future reader who might want to update the dependency.

Reply:
Good point, i've added a comment.